docs/website/docs/reference/tuning.md

```diff
@@ -0,0 +1,126 @@
+---
+icon: octicons/meter-16
+---
+
+# Tuning
+
+This page documents support for IREE dispatch tuning. The compiler supports
+both default and user-provided tuning specs that override compiler heuristics
+that guide dispatch code generation.
+
+The default specs are shipped with the IREE compiler and are target-specific.
+We aim to provide default tuning specs that cover the most in-demand hardware
+and dispatches from most popular ML models, although we do not guarantee
+completeness.
+
+User-provided tuning specs are a mechanism to cover out gaps in target hardware
+and model coverage.
+
+Currently, the dispatch tuner that generates tuning specs is still experimental
+and hosted
+[in an external repo](https://github.com/nod-ai/shark-ai/tree/main/tuner).
+
+## Flags
+
+Tuning specs are controlled with the following flags:
+
+* `--iree-codegen-enable-default-tuning-specs` -- enables or disabled the
+  default tuning specs shipped with the compiler.
```

Comment:
This could use some clarification... ideally with a diagram somehow how the components relate to one another.

You say that "the tuner that generates specs is in an external repo" then immediately afterwards say "use these flags". Those flags are for `iree-compile`, which _is_ in the iree-org/repo, right?

Reply:
I'm going to skip diagrams for now, but I reworded it a bit to make this part clearer